pep-0581.rst

```diff
@@ -0,0 +1,310 @@
+  REST API  [#roundup-rest-api]_. Last activity was in 2016.
+
+- I personally don't want to receive email each time someone added themselves as "nosy"
+  An issue has been filed in upstream roundup since 2012 with little traction [#nosy-list]_.
+
+- Creating an account has been a hassle. We've had reports when people have
+  trouble creating accounts or logging in.
+
+Why not GitLab
+--------------
+
+Had we migrated to GitLab instead of GitHub in 2017, this PEP would have been
+titled "Using GitLab Issues for CPython".
+
+Why not other issue tracker
+---------------------------
+
+Using another issue tracker will require yet another learning curve, of having
+to learn and get used to a different interface. We'll also need to learn and
+figure out how to build the integrations with GitHub.
+
+By using GitHub issues, where the source code is currently hosted, where pull requests
+are taking place, we'll be providing consistent experience to contributors and
+maintainers, and not having to jump from one interface to another.
+
+If someone else wants to do the research of what it takes for Python to start using
+other issue tracker, and what the migration plan is, please go ahead and come
+back with a PEP.
+
+Why not focus on improving Roundup / bpo
+----------------------------------------
+
+GitHub has many features we like that are already available. We still need to
+build out additional integrations and update our bots, but this is something
+we already know how to do.
+
+In order to really improve Roundup / bpo, it needs to first migrate to GitHub,
+add CI and bots. As I understand it, there is hesitation because upstream Roundup
+is still im mercurial. Someone more familiar with Roundup / bpo needs
+to champion this effort. (I'm not volunteering, I'm sorry).
+
+I believe the effort of creating and maintaining GitHub integrations and bots
+is much less than the effort needed to get Roundup up to speed and then to continue
+maintaining it.
+
+Migration Plan
+==============
+
+Backup of GitHub data
+---------------------
+
+This effort has been started and being tracked as an issue in core-workflow [#backup-github]_.
+We're using GitHub Migrations API [#gh-migrations-api]_ to download GitHub data
+for CPython on daily basis. The archives will be dropped in an S3 bucket.
+
+Thanks to Ernest W. Durbin III for working on this.
+
+Update the CLA host
+-------------------
+
+At the moment the CLA is hosted within bpo. It needs to be updated such that
+signing the CLA does not require a bpo account, and it should be hosted outside
+of the bpo.
+
+The current CLA process itself is not ideal. Currently contributors to
+devguide, peps, and core-workflow need to sign a CLA,  and it requires a bpo
+account. A bpo account should not be required for those projects.
+
+Currently the CLA process involves real human to manually check the records.
+Question: Will it be possible to completely automate the CLA process, so
```

Comment:
This would be a great time to start a thread with PSF legal on this. On the technical side, there are already great projects to handle this like http://clabot.github.io/

Reply:
I was thinking that this itself might require a separate PEP 😅